css/common.css

```diff
@@ -1,64 +1,60 @@
 @font-face {
-        font-family: "Spirax";
-        src: url("../css/Spirax-Regular.ttf")
+    font-family: "Spirax";
+    src: url("../css/Spirax-Regular.ttf")
 }
 
-body, ul, li, div, span, p, strong {
-  margin:0;
-  padding:0;
-  position:relative;
-}
```

Comment:
disclaimer: All those comments are me not being primarly a html/css designer. I almost never use relative/fixed.